apps/epic-web/src/templates/workshop-template.tsx

```diff
@@ -223,7 +224,7 @@ const WorkshopTemplate: React.FC<{
               )}
 
               {workshop && (
-                <Collection.Root module={workshop}>
+                <Collection.Root module={workshop as Module}>
                   <div className="flex w-full items-center justify-between pb-3">
```

Comment:
Library code is expecting a Module type, so I'm just casting Workshop to Module since Workshop is an extended Module